core/src/autogluon/core/searcher/bayesopt/tuning_algorithms/defaults.py

```diff
@@ -1,13 +1,9 @@
 from .bo_algorithm_components import LBFGSOptimizeAcquisition
-from ..models.meanstd_acqfunc import EIAcquisitionFunction
+from ..models.meanstd_acqfunc_impl import EIAcquisitionFunction
 
 
 DEFAULT_ACQUISITION_FUNCTION = EIAcquisitionFunction
 DEFAULT_LOCAL_OPTIMIZER_CLASS = LBFGSOptimizeAcquisition
 DEFAULT_NUM_INITIAL_CANDIDATES = 250
 DEFAULT_NUM_INITIAL_RANDOM_EVALUATIONS = 3
-DEFAULT_METRIC = 'active_metric'
```

Comment:
Is there a good reason for moving this to base_classes, and not leave it here?

I've never been a big fan of this defaults.py, but the intention was to collect constants here, and why not? Can't you import from base_classes?

Reply:
The reason is that EIAcquisitionfunction requires it, resulting in a circular import.